watchtower/blob/justice_kit.go

```diff
@@ -284,7 +286,9 @@ func Decrypt(key BreachKey, ciphertext []byte,
 
 	// If decryption succeeded, we will then decode the plaintext bytes
 	// using the specified blob version.
-	boj := &JusticeKit{}
+	boj := &JusticeKit{
+		BlobType: blobType,
+	}
```

Comment:
Don't we also need to update the `encode/decode` methods? As looks like they'll fail with an unknown blob type as is.

Reply:
nope, the encode/decode logic is gate on `FlagCommitOutputs` which is also set in `TypeAltruistAnchorCommit`. the payload format doesn't change at all between the legacy and anchor channel blobs